core/templates/pages/story-editor-page/story-editor-page.component.ts

```diff
@@ -35,19 +36,20 @@ require('pages/story-editor-page/services/story-editor-state.service.ts');
 
 require('pages/story-editor-page/story-editor-page.constants.ajs.ts');
 
+import { Subscription } from 'rxjs';
+
 angular.module('oppia').component('storyEditorPage', {
   template: require('./story-editor-page.component.html'),
   controller: [
     '$scope', '$uibModal', '$window', 'PageTitleService',
     'StoryEditorStateService', 'UndoRedoService',
     'UrlInterpolationService', 'UrlService',
-    'EVENT_STORY_INITIALIZED', 'EVENT_STORY_REINITIALIZED',
     function(
         $scope, $uibModal, $window, PageTitleService,
         StoryEditorStateService, UndoRedoService,
-        UrlInterpolationService, UrlService,
-        EVENT_STORY_INITIALIZED, EVENT_STORY_REINITIALIZED) {
+        UrlInterpolationService, UrlService) {
       var ctrl = this;
+      ctrl.attachedSubscriptions = new Subscription();
```

Comment:
Should we call this directiveSubscriptions?

Reply:
I don't know, I was just using it to chain the subscriptions so I don't have to unsubscribe to each one individually. We can follow a convention and stick to it for all files. I'm currently making a document for changes I made, so I can document this. Should I go ahead with directiveSubscriptions?

Reply:
> I don't know, I was just using it to chain the subscriptions so I don't have to unsubscribe to each one individually.

I understand that and we are already using this exact mechanism. Check line 63 in skill-questions-tab.directive.ts

Reply:
Ah, okay, then I'll change to directiveSubscriptions and add directly without declaring.

Reply:
Done